src/GenFacades/GenFacades.Console/GenFacades.Console.net46.csproj

```diff
@@ -4,25 +4,23 @@
 
   <PropertyGroup>
```

Comment:
Is keeping the net46 version in the same directory going to cause any issues? Hopefully not but before were were keeping it in a separate directory similar to all the other projects.